build.gradle

```diff
@@ -27,6 +27,7 @@ intellij {
 
 dependencies {
     compile 'io.fabric8:openshift-client:4.9.0'
+    compile 'io.fabric8:tekton-client:4.10.2'
```

Comment:
Please make sure we use a consistent version between OpenShift client and Tekton client